modules/ui/src/main/scala/lucuma/ui/reusability/package.scala

```diff
@@ -90,9 +91,10 @@ trait MathReusabilityInstances {
  * reusability of time types
  */
 trait TimeReusabilityInstances {
-  implicit val durationReuse: Reusability[Duration] = Reusability.by(_.getSeconds)
-  implicit val instantReuse: Reusability[Instant]   =
+  implicit val durationReuse: Reusability[Duration]   = Reusability.by(_.getSeconds)
```

Comment:
A `Duration` is made of `seconds` and `nano`. Should we include `getNano` in the mix here?

Reply:
Seems like it, although that isn't part of my change. 😆

Reply:
Reusability[Duration] is not used in Explore, so I removed it.